### PR TITLE
RI-5597: Fix Dependabot alerts

### PR DESCRIPTION
**Pull request by GnaneshKunal**
#### Bumps
- webpack-dev-middleware
- express (and all NestJS packages)
- follow-redirects
- IP

# Important

I noticed that platform-express used a fixed older version of the express package. While we can force install a specific express version via NPM resolutions, I have explored upgrading the platform-express package to bump the express package. I also took the opportunity to bump the rest of the NestJS packages. Also, I have removed the nest-router package, for which support exists in the existing NestJS library itself.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -84,8 +84,7 @@
     "word-wrap": "1.2.4",
     "electron-builder/app-builder-lib/@electron/universal/dir-compare/minimatch": "^3.0.5",
     "**/semver": "^7.5.2",
-    "rawproto/protobufjs": "^7.2.4",
-    "**/follow-redirects": "^1.15.4"
```
Comment (arthosofteq) [inline package.json:88]: @GnaneshKunal Are you sure that we must remove this line?
@zalenskiSofteq @romansergeenkosofteq Do you happy to know why we've added it? Is it safe to remove?

**Review by ArtemHoruzhenko — DISMISSED** [review]
I hope that router was the only 1 thing affected by bumping critical version of core core deps
from BE perspective looks good. Thanks!

let see what guys answer about removing follow-redirects

File: package.json (hunk 1 above)
Comment (GnaneshKunal) [reply package.json:88]: Yeah. Safe for removal. `follow-redirects` now has a new version released, `1.15.6`, which must be upgraded, but the NPM resolution has blocked the upgrade. So, it's necessary to remove this.

**Review by zalenskiSofteq — DISMISSED** [review]
fix conflict please

**Pull request merged**